src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java

```diff
@@ -146,11 +161,22 @@ public void onBuildStart(AbstractBuild<?, ?> build, TaskListener listener, GHRep
         } else {
             sb.append(Ghprb.replaceMacros(build, listener, startedStatus));
         }
-        createCommitStatus(build, listener, sb.toString(), repo, GHCommitState.PENDING);
+        logger.log(Level.FINE, "Full Display Name onBuildStart: " + build.getFullDisplayName() + " Attempting to send GitHub commit status");
+
+
+        if(!showMatrixStatus || (showMatrixStatus && build.getProject() instanceof MatrixProject)){
```

Comment:
I think you want to change the logic here to be IF project instanceof matrix project && !showMatrixStatus return?  Then if it is just a normal job we still get the pending state.   
```
if (build.getProject() instanceof MatrixProject && !showMatrixStatus) {
  return;
}
createCommitStatus(build, listener, sb.toString(), repo, GHCommitState.PENDING);
```